attachment_log/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Attachment Log",
+    "version": "16.0.1.0.0",
+    "author": "Cetmix, Odoo Community Association (OCA)",
+    "summary": "Add log info for attachments",
```

Comment:
You are not adding any log info to attachments. You are showing tracking information or metadata.
So should be something like `Show attachment  information in chatter`.